docs/machine-learning/tutorials/taxi-fare.md

```diff
@@ -39,75 +39,81 @@ This problem is centered around **predicting the fare of a taxi trip in New York
 
 To predict the taxi fare, you first select the appropriate machine learning task. You are looking to predict a real value (a double that represents price) based on the other factors in the dataset. You choose a [**regression**](../resources/glossary.md#regression) task.
 
-The process of training the model identifies which factors in the dataset are most influential when predicting the final fare price.
```

Comment:
I've removed that sentence because this is the gun on the wall that never shoots. What I mean is the tutorial never comes back to that topic again. And as the tutorial is already long, I thought it's not worth to keep the open ends in.